_wiki/sfmb_betatest_faq.md

```diff
@@ -56,6 +56,22 @@ latex   : false
 - Because game uses Discord Rich Presence to check your Discord ID.
 - It's best way to check user is REAL beta tester in server or not.
 
+### Q. Some game sprites are bugged, what should i do?
+- Game sprites got corrupted by some reason, mostly download issue. Try [This](sfmb://forceupdate)
+- Or put `sfmb://forceupdate` address on your browser. It should fix corrupted files.

```

Comment:
`put 'sfmb://forceupdate' address on your browser`
- Unnecessary solution
- It's same with link but difficult for most people